Content.Shared/_RMC14/Xenonids/Evolution/XenoEvolutionSystem.cs

```diff
@@ -219,8 +224,11 @@ private void OnXenoEvolveDoAfter(Entity<XenoEvolutionComponent> xeno, ref XenoEv
         _mind.TransferTo(mindId, newXeno);
         _mind.UnVisit(mindId);
 
-        // TODO RMC14 this is a hack because climbing on a newly created entity does not work properly for the client
-        var comp = EnsureComp<XenoNewlyEvolvedComponent>(newXeno);
+		foreach (var held in _hands.EnumerateHeld(xeno))
+			_hands.TryDrop(xeno, held);
+
+		// TODO RMC14 this is a hack because climbing on a newly created entity does not work properly for the client
+		var comp = EnsureComp<XenoNewlyEvolvedComponent>(newXeno);
```

Comment:
Somehow specifically these lines were tabs instead of spaces

```suggestion
        foreach (var held in _hands.EnumerateHeld(xeno))
            _hands.TryDrop(xeno, held);

        // TODO RMC14 this is a hack because climbing on a newly created entity does not work properly for the client
        var comp = EnsureComp<XenoNewlyEvolvedComponent>(newXeno);
```